tests/affiliatedcertification/affiliatedcerthelper/operator.go

```diff
@@ -193,8 +175,32 @@ func EnableCatalogSource(name string) error {
 	return setCatalogSource(false, name)
 }
 
-func IsCatalogSourceEnabled(name, namespace string) bool {
-	_, err := globalhelper.APIClient.CatalogSources(namespace).Get(context.TODO(), name, metav1.GetOptions{})
+func IsCatalogSourceEnabled(name, namespace, displayName string) (bool, error) {
+	source, err := globalhelper.APIClient.CatalogSources(namespace).Get(context.TODO(), name, metav1.GetOptions{})
+	if err != nil {
+		if k8serrors.IsNotFound(err) {
+			return false, nil
```

Comment:
This check is not needed, as the result is the same `return false, nil` as in line 185.